src/trusted/service_runtime/nacl_syscall_common.c

```diff
@@ -578,68 +620,93 @@ int32_t NaClSysOpen(struct NaClAppThread  *natp,
   retval = NaClHostDescStat(path, &stbuf);
 
   /* Windows does not have S_ISDIR(m) macro */
-  if (0 == retval && S_IFDIR == (S_IFDIR & stbuf.st_mode)) {
+  if (!retval && S_IFDIR == (S_IFDIR & stbuf.st_mode)) {
     struct NaClHostDir  *hd;
 
     hd = malloc(sizeof *hd);
-    if (NULL == hd) {
+    if (!hd) {
       retval = -NACL_ABI_ENOMEM;
       goto cleanup;
     }
     retval = NaClHostDirOpen(hd, path);
     NaClLog(1, "NaClHostDirOpen(0x%08"NACL_PRIxPTR", %s) returned %d\n",
             (uintptr_t) hd, path, retval);
-    if (0 == retval) {
-      retval = NaClSetAvail(nap,
-                            ((struct NaClDesc *) NaClDescDirDescMake(hd)));
-      NaClLog(1, "Entered directory into open file table at %d\n",
-              retval);
+    if (!retval) {
+      retval = NaClSetAvail(nap, ((struct NaClDesc *) NaClDescDirDescMake(hd)));
+      NaClLog(1, "Entered directory into open file table at %d\n", retval);
```

Comment:
Watch out for other uses of `enter` (i.e. where entering a function) in debug messages. Perhaps reword here.

Reply:
Yeah, I'll reword that.